data/remnant outfits.txt

```diff
@@ -342,6 +339,19 @@ outfit "Quantum Key Stone"
 	"quantum keystone" 1
 	description "This precious artifact attunes a ship's quantum oscillations in a way that allows it to travel through certain otherwise impassible wormholes in the Ember Waste. The stones are rare and valuable, because they can only be mined from one location in the Waste."
 
+outfit "Scanner Software"
+	category "Systems"
+	licenses
+		Remnant
+	cost 240000
+	thumbnail "outfit/scanner software"
```

Comment:
As MZ mentioned, 
>The "scanner software" sprite should really be a 3D render, rather than a completely different artistic style. 

And because of his other comment
>(And, I'm not sure if it makes sense to have outfits that are "software," instead of a physical item you install in the ship.)

it may be worth using a different name.

Reply:
I'd already change its name in the description to being a "software module," but sure, I'll change it all. How about "Salvage Scanner" ? (If you disagree, I can change it to something else, but in the mean time, I have changed it to this).

Likewise, when he made the comment the software scanner's icone was purely the code box, if I recall correctly. So it has at least transitioned halfway away from that. 

Best case, it is fine the way it is; worst case, it is a placeholder for a new graphic.

Reply:
Salvage Scanner works for me.